clang-tools-extra/test/clang-doc/enum.cpp

```diff
@@ -0,0 +1,38 @@
+// RUN: clang-doc --format=html --doxygen --output=%t/docs --executor=standalone %s
+// RUN: clang-doc --format=md --doxygen --output=%t/docs --executor=standalone %s
+// RUN: FileCheck %s -input-file=%t/docs/GlobalNamespace/index.html -check-prefix=HTML-INDEX
+// RUN: FileCheck %s -input-file=%t/docs/GlobalNamespace/index.md -check-prefix=MD-INDEX
+
+/**
+ * @brief For specifying RGB colors
+ */
+enum Color {
+  Red, // Red enums
+  Green, // Green enums
+  Blue // Blue enums
```

Comment:
should these be doc comments?

Reply:
These seem to be gone now, but isn’t it common to document the enum values? I know we do that in several places in LLVM. 

Thinking about other cases, it probably also worth having an “enum class”, an enum within a class, and one in a namespace.  I’d suggest also varying in at least one case where the doc comments are: e.g., above the value vs on the same line.